### ISSUE TITLE
GEM: Account for blade offsets in jaws

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#2996]
As a GEM scientist I would like the values set by the jaws manager to be as accurate as possible. Currently, IBEX assumes that both vertical and horizontal blades are in the same plane and calculates the jaws manager geometry based on this. They are in fact in slightly different planes (exact numbers to be confirmed).

**Comment by kjwoodsISIS** [comment]
Would it be possible to get a diagram (which we could attach to the wiki)  showing the geometry of the blades, so that we can relate the calculation of blade positions to the diagram.  I think that would help a lot in understanding the detailed calculations.

**Comment by DominicOram** [comment]
![jaws](https://user-images.githubusercontent.com/10086797/36558515-33037662-1803-11e8-8107-0ae1b68fbc3b.jpg)

My drawing skills are not great but does this help? As the width of the gap is a function of how 'far' along the beam you are the horizontal and vertical widths should be different. (Assuming x is non-negligible)

**Comment by ChrisM-S** [comment]
This would be good to also see on the OPI in the "Jaws Manager" view (in particular the side on view I mean) as the mimic is pretty good already. There is potentially space available in the device screen view(comparing with the POLARIS view of similar jaws).

Just one curiosity, would it not make more sense to show this view by default for jaws - also "Jaws manager" sounds like a more complex technical view rather than a clear visual which could help the more inexperienced user to see if the jaws might be - closed - for example?

**Comment by kjwoodsISIS** [comment]
@DominicOram Yes, this helps.  Thanks for compiling it.  However, I am not quite sure we have fully captured the physical situation we're trying to account for.
If the beam is assumed to travel horizontally (i.e. absolutely parallel to the floor), then I don't think the separation between the N-S and E-W blade pairs makes any difference to the calculation of the vertical & horizontal gaps.  On the other hand, if the beam travels at an angle to the horizontal (i.e. because neutrons have mass and fall (a small amount) under gravity), then the separation between the blades does make a difference.  
Is it the fall of the neutrons we are trying to take account of here?  Is the beam coming in at an angle from some other direction? (maybe the beam doesn't travel exactly parallel to the centre line?)  Or is there some other effect that I have missed completely?
I think I need to see the path of the beam, relative to the instrument centre line, in both side and plan (top) views to fully understand what effect we are trying to account for.  Perhaps we should put our heads together and draw a diagram we are both happy with?
